alert-common/src/main/java/com/synopsys/integration/alert/common/action/api/MultiResponseModel.java

```diff
@@ -0,0 +1,23 @@
+package com.synopsys.integration.alert.common.action.api;
+
+import java.util.List;
+
+import com.synopsys.integration.alert.common.rest.model.AlertSerializableModel;
+
+public class MultiResponseModel<T extends AlertSerializableModel> extends AlertSerializableModel {
+    private final List<T> models;
+
+    MultiResponseModel() {
+        // For serialization
+        this(List.of());
+    }
+
+    public MultiResponseModel(List<T> models) {
+        this.models = models;
+    }
+
+    public List<T> getModels() {
```

Comment:
You'll need to annotate this method with `JsonIgnore` so that the JSON object created doesn't add a models attribute to the resulting object.

Reply:
Could I alternatively (or in addition) make the method protected?

Reply:
I honestly not sure what Jackson would do.  From what I understand it uses the getters and setters which are usually public but I don't know if protected getter would be serialized or hidden.  I assume it would be hidden.

Reply:
https://github.com/FasterXML/jackson-docs

Reply:
Regardless of Jackson, I think `getModels()` should be protected to better express intent.